src/Board.py

```diff
@@ -50,6 +51,85 @@ def __init__(self, style: str = "default", size: int = 5) -> None:
         else:
             raise ValueError("Unrecognised map style")
 
+    def get(self, coords):
+        return self.data[coords[1]][coords[0]]
+
+    def get_directly_adj(self, start=(0, 0), terain_filter=None) -> list:
```

Comment:
```suggestion
    def get_directly_adj(
        self,
        start: Coords = (0, 0),
        terrain_filter: Terrain | None = None,
    ) -> List[Coords]:
```

Mostly adding typing but also `terain` -> `terrain`